api/types/constants.go

```diff
@@ -1174,3 +1174,8 @@ const (
 	// installer script when agentless mode is enabled for a matcher
 	DefaultInstallerScriptNameAgentless = "default-agentless-installer"
 )
+
+const (
+	ProtocolHTTP = "HTTP"
+	ProtocolTCP  = "TCP"
+)
```

Comment:
nit: i think it's better to be specific what protocol it is for (e.g `ApplicationProtocolHTTP`. Also need some godoc. db protocols for ref:
https://github.com/gravitational/teleport/blob/01a62eb57f32faac0866d22fa269141c7e7d7a00/api/types/database.go#L1040-L1044